cmd/btcatomicswap/main.go

```diff
@@ -456,25 +456,52 @@ func fundRawTransaction(c *rpc.Client, tx *wire.MsgTx, feePerKb btcutil.Amount)
 // wallet.  If unset, it attempts to find an estimate using estimatefee 6.  If
 // both of these fail, it falls back to mempool relay fee policy.
 func getFeePerKb(c *rpc.Client) (useFee, relayFee btcutil.Amount, err error) {
-	info, err := c.GetInfo()
+	var raw map[string]interface{}
```

Comment:
Instead of unmarshalling into a map like this, a structure is safer and more readable:

```
var networkInfo struct {
    RelayFee float64 `json:"relayfee"`
}
err = json.Unmarshal(jsonNetworkInfo, &networkInfo)
if err != nil {
    // ...
}
// use networkInfo.RelayFee
```

The extra fields will be ignored since they aren't in the struct.

Reply:
I see, my first time working with go and couldnt get it to work with a struct.  A lot nicer like that.